Readme.md

```diff
@@ -1,90 +1,179 @@
+# API Design Style Guide Contracts [![CircleCI](https://circleci.com/gh/apiaryio/constitutions.svg?style=shield&circle-token=32b6b19a59e02fdb795d2f47c80e37e2a2a9a033&ver=1)](https://circleci.com/gh/apiaryio/constitutions)
 
-> **Let's define API Style Guides collaboratvelly on examples.** 
+> **Let's define API Style Guides collaboratively on examples.** 
 
-This repository containts executable examples of API Design Style Guide Rules (API Blueprint, OpeAPI/Swagger, ... code snippets) for test-driven development of API Description Language linters (style guide engines) to enable vendor indepndence, all the time up-to-date documentation tested in CI and preventing regressions caused by unwanted changes in behavior of underying Style Guide engines and linters implementation.
+This repository contains executable examples of API Design Style Guide Rules (API Blueprint, OpeAPI/Swagger, ... code snippets) to enable vendor independence, all the time up-to-date documentation tested in CI and preventing regressions caused by unwanted changes in behavior of underlying Style Guide engine.
 
 ## How to write an API Design Style Guide
 
 ### 1. Prepare the Style Guide concepually
 - Catch the idea
 - Create a written free-form verbal documentaion, formal specification, white paper etc..
-- Collect all the PDFs, Google Docs, Markdowns, READMEs, rtfs and and docxs you already have in your drawers
+- Collect all the PDFs, Google Docs, Markdowns, READMEs, rtfs and and docs you already have in your drawers
 
 ### 2. Breakdown the specification into single rules
-- Identify and isolate singel rules in the in the textual styleguide you created in the step 1
+- Identify and isolate single rules in the in the textual styleguide you created in the step 1
 - Add good and bad examples for every single new rule using API Description language (OAS/Swagger, API Blueprint, ...)
 
-### 3. TDD of the rules, add your Style Guide to CI
-- Hook-up an API Description language Style Guide engine in and execute examples against Rules assertion code as fixtures in your tests (see: `/styleguide/sample-styleguide/linter/linter`
+### 3. Development of the rules
 
-- Make the linter **pass** on all **good examples**
-- Make the linter **fail** on all **bad examples**
+#### 3.1 Install prerequisites
 
+- install [`node.js` v6+](https://nodejs.org/en/download/)
+- install dependencies
+
+```bash
+$ npm install
 ```
-$ ./scripts/test
-Testing styleguide 'sample-styleguide'
-  Testing rule 'sample-rule'
-    Expecting good example 'sample-good-example' to pass
-      PASS
-    Expecting bad example 'sample-bad-example' to fail
-      PASS
+
+#### 3.2 Create new styleguide rule directory structure and files
+
+Fork this [repo](https://github.com/apiaryio/constitutions)
+
+Run:
+
 ```
+$ ./scripts/init name-of-your-new-rule-directory
```

Comment:
Extra line is wrongly formatted in readme on github

Reply:
fixed